### ISSUE TITLE
Persisting data to skip setup

### ISSUE THREAD

**Issue by Dids** [CachetHQ/Docker#131]
I'm running Cachet using Docker Cloud, which uses a similar syntax to Docker Compose files, but it's not identical.

My question is, how would I make sure that I can persist data in a volume, as well as pass the correct environment variables, so that I could replicate an identical Cachet setup at any point, as long as my data (including database) was still there?

Essentially, I'd like to know what paths on Cachet's side I'd have to mount (if any), and what env vars I'd have to define to skip the initial setup. I know that at least previously I had to define APP_KEY etc. but it seems like the format/encryption for that has changed?

**Comment by Dids** [comment]
Actually, from what I can tell, I no longer need to set APP_KEY, from what I can tell. I'm currently setting the following env vars:

```
- APP_ENV=development
- APP_DEBUG=true
- APP_URL=http://url
- DB_DRIVER=mysql
- DB_HOST=mysql
- DB_DATABASE=cachet
- DB_USERNAME=cachet
- DB_PASSWORD=pass
- CACHE_DRIVER=apc
- SESSION_DRIVER=apc
- QUEUE_DRIVER=database
- CACHET_EMOJI=true
- MAIL_DRIVER=smtp
- MAIL_HOST=mailtrap.io
- MAIL_PORT=2525
- MAIL_USERNAME=user
- MAIL_PASSWORD=pass
- MAIL_ADDRESS="no-reply@url"
- MAIL_NAME="Status"
- MAIL_ENCRYPTION=tls
- REDIS_HOST=null
- REDIS_DATABASE=null
- REDIS_PORT=null
- GITHUB_TOKEN=null
```

**Comment by djdefi** [comment]
That is a good question. I messed with this a bit but was not able to figure out how to skip the setup wizards.

**Comment by djdefi** [comment]
Possible fix in #166

**Comment by djdefi** [comment]
I am going to go ahead and close this, as #166 should make this work better.
